cpu/stm32/kconfigs/Kconfig.clk

```diff
@@ -90,7 +147,7 @@ config CLOCK_APB1_DIV
     default 16 if CLOCK_APB1_DIV_16
 
 choice
-bool "APB2 prescaler (division factor of HCLK to produce PCLK2)"
+bool "APB2 prescaler (division factor of HCLK to produce PCLK2)"  if CPU_FAM_G4
```

Comment:
This causes no choice to be selected when G0 family is used, so `CLOCK_APB2_DIV` does not get a value, is this behaviour intended?

Reply:
There's no APB2 on G0 so yes, this is intended.

Reply:
Ok, thanks for clarifying!